saleor/plugins/webhook/plugin.py

```diff
@@ -2995,30 +2995,89 @@ def process_payment(
             **kwargs,
         )
 
-    def get_taxes_for_checkout(
-        self, checkout_info, lines, previous_value
-    ) -> Optional["TaxData"]:
-        return trigger_all_webhooks_sync(
-            WebhookEventSyncType.CHECKOUT_CALCULATE_TAXES,
-            lambda: generate_checkout_payload_for_tax_calculation(
-                checkout_info,
-                lines,
-            ),
-            parse_tax_data,
-            checkout_info.checkout,
+    def __run_tax_webhook(
+        self,
+        event_type: str,
+        app_identifier: str,
+        payload: str,
+        subscriptable_object=None,
+    ):
+        app = (
+            App.objects.filter(
```

Comment:
Can't you provide `app_identifier` to `get_webhooks_for_event` to get the same result?